timesketch/frontend/src/views/Intelligence.vue

```diff
@@ -59,53 +186,116 @@ limitations under the License.
 
 <script>
 import ApiClient from '../utils/RestApiClient'
-import _ from 'lodash'
-
-import TsDynamicTable from '../components/Common/TsDynamicTable'
+import { SnackbarProgrammatic as Snackbar } from 'buefy'
 
 export default {
-  components: {
-    TsDynamicTable,
-  },
   data() {
     return {
-      localIntelligenceMeta: {
-        columns: [
-          {
-            field: 'ioc',
-            label: 'IOC',
-            searchable: true,
-            type: 'text',
-          },
-          {
-            field: 'type',
-            label: 'Type',
-            searchable: false,
-            type: 'text',
-          },
-        ],
-        key: 'local_intel',
-        label: 'Local intelligence',
-        deletable: true,
-      },
+      sketchTags: [],
+      tagInfo: {},
+      editingIoc: {},
+      showEditModal: false,
+      IOCTypes: [
+        { regex: /^(\/[\S]+)+$/i, type: 'fs_path' },
+        { regex: /^([-\w]+\.)+[a-z]{2,}$/i, type: 'hostname' },
+        { regex: /^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$/g, type: 'ip' },
```

Comment:
Seeing this is v4 only, maybe name it `ipv4`?

Reply:
Done